### ISSUE TITLE
Improvements to bootswatch materia styles

### ISSUE THREAD

**Issue by cpsievert** [rstudio/bslib#148]
```r
library(bootstraplib)
bs_theme_preview(bs_theme(bootswatch = "materia"))
```

<img width="1271" alt="Screen Shot 2020-11-03 at 2 29 38 PM" src="https://user-images.githubusercontent.com/1365941/98041324-5a9fb900-1de7-11eb-8023-57fc3e4ed664.png">

* When it comes to `selectInput()`, the missing "border" is a result of materia inlining box-shadow rules without setting the proper Sass variable defaults.

* When it comes to `sliderInput()` transparent handles, this is because materia sets `$input-bg: transparent`, which would do a better job at accounting for in Shiny itself (https://github.com/rstudio/shiny/pull/3131)

**Comment by cpsievert** [comment]
This is a result of materia inlining box-shadow rules without setting the proper Sass variable defaults 😞

**Comment by cpsievert** [comment]
Seems the sliderInput() transparency issue is best addressed in shiny (https://github.com/rstudio/shiny/pull/3131)

**Comment by cpsievert** [comment]
Closed via #155
